### DISCUSSION TITLE
Relative path for pasted image directory?

### DISCUSSION THREAD

**Discussion by rudolphmax** [notaapp/talk#395] [Feature Requests]
Is it possible to provide support for a relative path in the `Where to save pasted images and files` setting? 

For example, my workflow usually involves separate folders for different subjects, clients, ..., each with a dedicated `assets/` folder. For me, it's important to keep media separate - especially when sharing data, as I often store sensitive information and files I can't distribute with every markdown file I share.

**Comment by astoilkov** [comment]
Yes, I actually also need it. It would be a useful addition. Thanks.

**Comment by lolbat** [comment]
Many static site generators use a single directory to store images for blog posts and pages. For instance, Hugo stores images in the `/static/images` and references them in your posts as `/images/`.

So it would be very handy to have a setting that would let you indicate a directory to store the images and also a base path to reference them from in your note.

**Reply by astoilkov** [reply]
Do you mean that in Hugo, if you write `/static/images/image.png` this won't work?

**Comment by lolbat** [comment]
It won't. The site serves the images from the directory under the `/static/` directory. So it would be `/images/image.png`. The markdown would also have to know that the link was different from the path when it went to render as well.

IIRC this is a feature that Typora has.
